src/trace/segment.ts

```diff
@@ -48,8 +48,8 @@ export default function traceSegment(options: CustomOptionsType) {
       url = new URL(window.location.href);
       url.pathname = config[1];
     }
-    if (Array.isArray(options.originAllowList)) {
-      const traced = options.originAllowList.some((rule) => {
+    if (Array.isArray(options.noTraceOrigins)) {
```

Comment:
Could you add default value(`[]`) for `noTraceOrigins` to remove this `if`.

Reply:
👌